<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!--NewPage-->
<HTML>
<HEAD>
<!-- Generated by javadoc (build 1.5.0_22) on Fri Oct 22 10:46:53 GMT-05:00 2010 -->
<TITLE>
ProteinSequence (Parallel Java Library Documentation)
</TITLE>

<META NAME="keywords" CONTENT="edu.rit.compbio.seq.ProteinSequence class">

<LINK REL ="stylesheet" TYPE="text/css" HREF="../../../../stylesheet.css" TITLE="Style">

<SCRIPT type="text/javascript">
function windowTitle()
{
    parent.document.title="ProteinSequence (Parallel Java Library Documentation)";
}
</SCRIPT>
<NOSCRIPT>
</NOSCRIPT>

</HEAD>

<BODY BGCOLOR="white" onload="windowTitle();">


<!-- ========= START OF TOP NAVBAR ======= -->
<A NAME="navbar_top"><!-- --></A>
<A HREF="#skip-navbar_top" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_top_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/ProteinSequence.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../index-files/index-1.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../../../edu/rit/compbio/seq/ProteinLocalAlignmentSmp.html" title="class in edu.rit.compbio.seq"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../../../edu/rit/compbio/seq/Sequence.html" title="class in edu.rit.compbio.seq"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../../../index.html?edu/rit/compbio/seq/ProteinSequence.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="ProteinSequence.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_top"></A>
<!-- ========= END OF TOP NAVBAR ========= -->

<HR>
<!-- ======== START OF CLASS DATA ======== -->
<H2>
<FONT SIZE="-1">
edu.rit.compbio.seq</FONT>
<BR>
Class ProteinSequence</H2>
<PRE>
<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html" title="class or interface in java.lang">java.lang.Object</A>
  <IMG SRC="../../../../resources/inherit.gif" ALT="extended by "><A HREF="../../../../edu/rit/compbio/seq/Sequence.html" title="class in edu.rit.compbio.seq">edu.rit.compbio.seq.Sequence</A>
      <IMG SRC="../../../../resources/inherit.gif" ALT="extended by "><B>edu.rit.compbio.seq.ProteinSequence</B>
</PRE>
<HR>
<DL>
<DT><PRE>public class <B>ProteinSequence</B><DT>extends <A HREF="../../../../edu/rit/compbio/seq/Sequence.html" title="class in edu.rit.compbio.seq">Sequence</A></DL>
</PRE>

<P>
Class ProteinSequence encapsulates a protein sequence. A protein sequence
 object may be constructed from a string or read from a file. Protein sequence
 objects may also be read from a protein sequence database using class
 <A HREF="../../../../edu/rit/compbio/seq/ProteinDatabase.html" title="class in edu.rit.compbio.seq">ProteinDatabase</A>.
 <P>
 In a file, a protein sequence is stored in FASTA format. A FASTA format
 protein sequence consists of one description line and one or more sequence
 lines. The description line consists of an initial <TT>'&gt;'</TT> character
 followed by zero or more characters (the protein's description). A sequence
 line consists of one or more characters <TT>'A'</TT> through <TT>'Z'</TT>,
 <TT>'a'</TT> through <TT>'z'</TT>, <TT>'*'</TT>, or <TT>'-'</TT>.
 <P>
 In a program, a protein sequence is represented as a byte array (type
 <TT>byte[]</TT>). For a protein sequence of length <I>L</I>, the byte array
 contains <I>L</I>+1 bytes. The byte at index 0 is unused and contains a value
 of -1. The bytes at indexes 1 through <I>L</I> contain the amino acids. Amino
 acids <TT>'A'</TT> through <TT>'Z'</TT> (case insensitive) are represented by
 the values 0 through 25; <TT>'*'</TT> is represented as 26; <TT>'-'</TT> is
 represented as 27.
 <P>
 The amino acid letters and values are:
 <TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0>
 <TR><TD><I>Letter</I>&nbsp;&nbsp;</I></TD><TD><I>Value</I>&nbsp;&nbsp;</TD><TD><I>Amino Acid</I></TD></TR>
 <TR><TD>A</TD><TD>0</TD><TD>Alanine</TD></TR>
 <TR><TD>B</TD><TD>1</TD><TD>Aspartate or asparagine</TD></TR>
 <TR><TD>C</TD><TD>2</TD><TD>Cysteine</TD></TR>
 <TR><TD>D</TD><TD>3</TD><TD>Aspartate</TD></TR>
 <TR><TD>E</TD><TD>4</TD><TD>Glutamate</TD></TR>
 <TR><TD>F</TD><TD>5</TD><TD>Phenylalanine</TD></TR>
 <TR><TD>G</TD><TD>6</TD><TD>Glycine</TD></TR>
 <TR><TD>H</TD><TD>7</TD><TD>Histidine</TD></TR>
 <TR><TD>I</TD><TD>8</TD><TD>Isoleucine</TD></TR>
 <TR><TD>J</TD><TD>9</TD><TD><I>unused</I></TD></TR>
 <TR><TD>K</TD><TD>10</TD><TD>Lysine</TD></TR>
 <TR><TD>L</TD><TD>11</TD><TD>Leucine</TD></TR>
 <TR><TD>M</TD><TD>12</TD><TD>Methionine</TD></TR>
 <TR><TD>N</TD><TD>13</TD><TD>Asparagine</TD></TR>
 <TR><TD>O</TD><TD>14</TD><TD><I>unused</I></TD></TR>
 <TR><TD>P</TD><TD>15</TD><TD>Proline</TD></TR>
 <TR><TD>Q</TD><TD>16</TD><TD>Glutamine</TD></TR>
 <TR><TD>R</TD><TD>17</TD><TD>Arginine</TD></TR>
 <TR><TD>S</TD><TD>18</TD><TD>Serine</TD></TR>
 <TR><TD>T</TD><TD>19</TD><TD>Threonine</TD></TR>
 <TR><TD>U</TD><TD>20</TD><TD>Selenocysteine</TD></TR>
 <TR><TD>V</TD><TD>21</TD><TD>Valine</TD></TR>
 <TR><TD>W</TD><TD>22</TD><TD>Tryptophan</TD></TR>
 <TR><TD>X</TD><TD>23</TD><TD>Any, unknown</TD></TR>
 <TR><TD>Y</TD><TD>24</TD><TD>Tyrosine</TD></TR>
 <TR><TD>Z</TD><TD>25</TD><TD>Glutamate or glutamine</TD></TR>
 <TR><TD>*</TD><TD>26</TD><TD>Translation stop</TD></TR>
 <TR><TD>-</TD><TD>27</TD><TD>Gap of indeterminate length</TD></TR>
 </TABLE>
<P>

<P>
<HR>

<P>

<!-- ======== CONSTRUCTOR SUMMARY ======== -->

<A NAME="constructor_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Constructor Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../../../edu/rit/compbio/seq/ProteinSequence.html#ProteinSequence(java.io.File)">ProteinSequence</A></B>(<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/io/File.html" title="class or interface in java.io">File</A>&nbsp;file)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Construct a new protein sequence read from the given file.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><B><A HREF="../../../../edu/rit/compbio/seq/ProteinSequence.html#ProteinSequence(java.lang.String, java.lang.String)">ProteinSequence</A></B>(<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/String.html" title="class or interface in java.lang">String</A>&nbsp;description,
                <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/String.html" title="class or interface in java.lang">String</A>&nbsp;sequence)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Construct a new protein sequence from the given string.</TD>
</TR>
</TABLE>
&nbsp;
<!-- ========== METHOD SUMMARY =========== -->

<A NAME="method_summary"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="2"><FONT SIZE="+2">
<B>Method Summary</B></FONT></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;char</CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../../edu/rit/compbio/seq/ProteinSequence.html#charAt(int)">charAt</A></B>(int&nbsp;i)</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Returns a character version of this protein sequence's element at the
 given index.</TD>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD ALIGN="right" VALIGN="top" WIDTH="1%"><FONT SIZE="-1">
<CODE>&nbsp;<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/String.html" title="class or interface in java.lang">String</A></CODE></FONT></TD>
<TD><CODE><B><A HREF="../../../../edu/rit/compbio/seq/ProteinSequence.html#toString()">toString</A></B>()</CODE>

<BR>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Returns a string version of this protein sequence.</TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_edu.rit.compbio.seq.Sequence"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class edu.rit.compbio.seq.<A HREF="../../../../edu/rit/compbio/seq/Sequence.html" title="class in edu.rit.compbio.seq">Sequence</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="../../../../edu/rit/compbio/seq/Sequence.html#description()">description</A>, <A HREF="../../../../edu/rit/compbio/seq/Sequence.html#elementsToString()">elementsToString</A>, <A HREF="../../../../edu/rit/compbio/seq/Sequence.html#equals(java.lang.Object)">equals</A>, <A HREF="../../../../edu/rit/compbio/seq/Sequence.html#hashCode()">hashCode</A>, <A HREF="../../../../edu/rit/compbio/seq/Sequence.html#length()">length</A>, <A HREF="../../../../edu/rit/compbio/seq/Sequence.html#sequence()">sequence</A></CODE></TD>
</TR>
</TABLE>
&nbsp;<A NAME="methods_inherited_from_class_java.lang.Object"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#EEEEFF" CLASS="TableSubHeadingColor">
<TH ALIGN="left"><B>Methods inherited from class java.lang.<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html" title="class or interface in java.lang">Object</A></B></TH>
</TR>
<TR BGCOLOR="white" CLASS="TableRowColor">
<TD><CODE><A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#clone()" title="class or interface in java.lang">clone</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#finalize()" title="class or interface in java.lang">finalize</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#getClass()" title="class or interface in java.lang">getClass</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#notify()" title="class or interface in java.lang">notify</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#notifyAll()" title="class or interface in java.lang">notifyAll</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#wait()" title="class or interface in java.lang">wait</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#wait(long)" title="class or interface in java.lang">wait</A>, <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#wait(long, int)" title="class or interface in java.lang">wait</A></CODE></TD>
</TR>
</TABLE>
&nbsp;
<P>

<!-- ========= CONSTRUCTOR DETAIL ======== -->

<A NAME="constructor_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Constructor Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="ProteinSequence(java.lang.String, java.lang.String)"><!-- --></A><H3>
ProteinSequence</H3>
<PRE>
public <B>ProteinSequence</B>(<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/String.html" title="class or interface in java.lang">String</A>&nbsp;description,
                       <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/String.html" title="class or interface in java.lang">String</A>&nbsp;sequence)</PRE>
<DL>
<DD>Construct a new protein sequence from the given string.
<P>
<DL>
<DT><B>Parameters:</B><DD><CODE>description</CODE> - Description string. Must start with a <TT>'&gt;'</TT> character.<DD><CODE>sequence</CODE> - Sequence string. Must consist of the characters <TT>'A'</TT> through
     <TT>'Z'</TT>, <TT>'a'</TT> through <TT>'z'</TT>, <TT>'*'</TT>, and
     <TT>'-'</TT>.</DL>
</DL>
<HR>

<A NAME="ProteinSequence(java.io.File)"><!-- --></A><H3>
ProteinSequence</H3>
<PRE>
public <B>ProteinSequence</B>(<A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/io/File.html" title="class or interface in java.io">File</A>&nbsp;file)
                throws <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/io/IOException.html" title="class or interface in java.io">IOException</A></PRE>
<DL>
<DD>Construct a new protein sequence read from the given file.
<P>
<DL>
<DT><B>Parameters:</B><DD><CODE>file</CODE> - File.
<DT><B>Throws:</B>
<DD><CODE><A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/io/IOException.html" title="class or interface in java.io">IOException</A></CODE> - Thrown if an I/O error occurred.</DL>
</DL>

<!-- ============ METHOD DETAIL ========== -->

<A NAME="method_detail"><!-- --></A>
<TABLE BORDER="1" WIDTH="100%" CELLPADDING="3" CELLSPACING="0" SUMMARY="">
<TR BGCOLOR="#CCCCFF" CLASS="TableHeadingColor">
<TH ALIGN="left" COLSPAN="1"><FONT SIZE="+2">
<B>Method Detail</B></FONT></TH>
</TR>
</TABLE>

<A NAME="charAt(int)"><!-- --></A><H3>
charAt</H3>
<PRE>
public char <B>charAt</B>(int&nbsp;i)</PRE>
<DL>
<DD>Returns a character version of this protein sequence's element at the
 given index.
<P>
<DD><DL>
<DT><B>Specified by:</B><DD><CODE><A HREF="../../../../edu/rit/compbio/seq/Sequence.html#charAt(int)">charAt</A></CODE> in class <CODE><A HREF="../../../../edu/rit/compbio/seq/Sequence.html" title="class in edu.rit.compbio.seq">Sequence</A></CODE></DL>
</DD>
<DD><DL>
<DT><B>Parameters:</B><DD><CODE>i</CODE> - Index in the range 1 .. <I>L</I>.
<DT><B>Returns:</B><DD>Character corresponding to element <TT>i</TT>.</DL>
</DD>
</DL>
<HR>

<A NAME="toString()"><!-- --></A><H3>
toString</H3>
<PRE>
public <A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/String.html" title="class or interface in java.lang">String</A> <B>toString</B>()</PRE>
<DL>
<DD>Returns a string version of this protein sequence. The string is
 <TT>"ProteinSequence(<I>description</I>)"</TT>.
<P>
<DD><DL>
<DT><B>Overrides:</B><DD><CODE><A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html#toString()" title="class or interface in java.lang">toString</A></CODE> in class <CODE><A HREF="http://java.sun.com/j2se/1.5.0/docs/api/java/lang/Object.html" title="class or interface in java.lang">Object</A></CODE></DL>
</DD>
<DD><DL>

<DT><B>Returns:</B><DD>String version.</DL>
</DD>
</DL>
<!-- ========= END OF CLASS DATA ========= -->
<HR>


<!-- ======= START OF BOTTOM NAVBAR ====== -->
<A NAME="navbar_bottom"><!-- --></A>
<A HREF="#skip-navbar_bottom" title="Skip navigation links"></A>
<TABLE BORDER="0" WIDTH="100%" CELLPADDING="1" CELLSPACING="0" SUMMARY="">
<TR>
<TD COLSPAN=2 BGCOLOR="#EEEEFF" CLASS="NavBarCell1">
<A NAME="navbar_bottom_firstrow"><!-- --></A>
<TABLE BORDER="0" CELLPADDING="0" CELLSPACING="3" SUMMARY="">
  <TR ALIGN="center" VALIGN="top">
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../overview-summary.html"><FONT CLASS="NavBarFont1"><B>Overview</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-summary.html"><FONT CLASS="NavBarFont1"><B>Package</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#FFFFFF" CLASS="NavBarCell1Rev"> &nbsp;<FONT CLASS="NavBarFont1Rev"><B>Class</B></FONT>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="class-use/ProteinSequence.html"><FONT CLASS="NavBarFont1"><B>Use</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="package-tree.html"><FONT CLASS="NavBarFont1"><B>Tree</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../deprecated-list.html"><FONT CLASS="NavBarFont1"><B>Deprecated</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../index-files/index-1.html"><FONT CLASS="NavBarFont1"><B>Index</B></FONT></A>&nbsp;</TD>
  <TD BGCOLOR="#EEEEFF" CLASS="NavBarCell1">    <A HREF="../../../../help-doc.html"><FONT CLASS="NavBarFont1"><B>Help</B></FONT></A>&nbsp;</TD>
  </TR>
</TABLE>
</TD>
<TD ALIGN="right" VALIGN="top" ROWSPAN=3><EM>
</EM>
</TD>
</TR>

<TR>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
&nbsp;<A HREF="../../../../edu/rit/compbio/seq/ProteinLocalAlignmentSmp.html" title="class in edu.rit.compbio.seq"><B>PREV CLASS</B></A>&nbsp;
&nbsp;<A HREF="../../../../edu/rit/compbio/seq/Sequence.html" title="class in edu.rit.compbio.seq"><B>NEXT CLASS</B></A></FONT></TD>
<TD BGCOLOR="white" CLASS="NavBarCell2"><FONT SIZE="-2">
  <A HREF="../../../../index.html?edu/rit/compbio/seq/ProteinSequence.html" target="_top"><B>FRAMES</B></A>  &nbsp;
&nbsp;<A HREF="ProteinSequence.html" target="_top"><B>NO FRAMES</B></A>  &nbsp;
&nbsp;<SCRIPT type="text/javascript">
  <!--
  if(window==top) {
    document.writeln('<A HREF="../../../../allclasses-noframe.html"><B>All Classes</B></A>');
  }
  //-->
</SCRIPT>
<NOSCRIPT>
  <A HREF="../../../../allclasses-noframe.html"><B>All Classes</B></A>
</NOSCRIPT>


</FONT></TD>
</TR>
<TR>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
  SUMMARY:&nbsp;NESTED&nbsp;|&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_summary">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_summary">METHOD</A></FONT></TD>
<TD VALIGN="top" CLASS="NavBarCell3"><FONT SIZE="-2">
DETAIL:&nbsp;FIELD&nbsp;|&nbsp;<A HREF="#constructor_detail">CONSTR</A>&nbsp;|&nbsp;<A HREF="#method_detail">METHOD</A></FONT></TD>
</TR>
</TABLE>
<A NAME="skip-navbar_bottom"></A>
<!-- ======== END OF BOTTOM NAVBAR ======= -->

<HR>
<FONT SIZE="-2">Copyright &copy; 2005-2010 by Alan Kaminsky. All rights reserved. Send comments to ark</FONT><FONT SIZE="1" COLOR="#FFFFFF">&shy;</FONT><FONT SIZE="-2">&#64;</FONT><FONT SIZE="1" COLOR="#FFFFFF">&shy;</FONT><FONT SIZE="-2">cs.rit.edu.</FONT>
</BODY>
</HTML>
